src/docs/patterns/overlay.mdx

```diff
@@ -71,25 +77,35 @@ The modal dialog appears in the center of the screen in front of the primary con
 
 Simple modals should have a title, a descriptive message, and action buttons for users to respond to the prompt or dismiss the dialog. These modals should be no larger than is necessary to contain their contents.
 
+<ImageGrid
+    images={[DialogText1, DialogText2, DialogText3, DialogText4, DialogText5, DialogText6]}
+    captionsUnderImages={[
+        'DONT: Do not arrange button so that your users are more likely to click the destructive option.',
```

Comment:
This picture doesn't really seem to be showing 'arrangement' of buttons like the caption says. This seems more like "Don't use normal button styles for destructive actions."